### ISSUE TITLE
Provide high contrast theme to widgets

### ISSUE THREAD

**Issue by Fox32** [vector-im/element-web#22631]
#### What would you like to do?

We would like to match the theme of our widget to the Element theme in case the high contrast theme is used.

#### Why would you like to do it?

We currently have widgets that try to look similar to Element by evaluating the theme passed to the widget (via `$org.matrix.msc2873.client_theme`, see https://github.com/matrix-org/matrix-spec-proposals/pull/2873).
The contrast of the Element light theme is not suitable to reach something like WCAG 2.0, but Element provides a high contrast theme (or theme option) that is. We want to make our widgets more accessible by providing a high contrast theme, too.

#### How would you like to achieve it?

`$org.matrix.msc2873.client_theme` should include whether the high contrast theme is used (see https://github.com/matrix-org/matrix-react-sdk/blob/develop/src/stores/widgets/StopGapWidget.ts#L126-L132). Right now the value is either `'light'` or `'dark'`. If the high contrast mode is used, it should be included in that value. The naming is not defined by MSC2873, so we could change it. On the other hand, it is a breaking change for existing widgets that assume a value here.

### Have you considered any alternatives?

As an alternative we are currently considering to use a high contrast theme by default for our widget and not to match the styling of Element.

**Comment by Fox32** [comment]
This seems to be resolved now. It is passing `light`, `dark` and `light-high-contrast`.
